build/Targets/VSL.Imports.targets

```diff
@@ -270,6 +270,23 @@
     </NuGetPackageToIncludeInVsix>
   </ItemDefinitionGroup>
 
+  <!-- Currently, the Willow installer can't install vsixes that have a <Prerequisites> element
+       but the Dev15 RC SDK packages can NOT install vsixes without them as part of the build.
+       To work around this conflict, we include them in our manifest files, but if we're building
+       with something older than Dev15 RC (as we do for official builds that we insert into setup)
```

Comment:
I think there is not just no plan, but a stated position that the engineering team doesn't want to have pre-release builds on these machines.  I think this is fine.  I also considered using `OfficialBuild=true`, which we could switch to if the machines *were* upgraded before Willow is ready to support this.